### ISSUE TITLE
what draft? schema field is missing

### ISSUE THREAD

**Issue by tommyjcarpenter** [devongovett/protobuf-jsonschema#12]
Converting a .proto file to json using this tool creates a list of definitions, but none of them have the `$schema$` field. This is a problem, because then you don't know what version of JSONschema the schema is, and you can't validate it against it. 

In the short term however, what draft of jsonschema does this tool generate out?

**Comment by tomdeering-wf** [comment]
Agree, you can't do much with a json-schema if you don't know what version of the specification to use.
